Sources/Anthropic/Private/Network/Endpoint.swift

```diff
@@ -27,12 +27,12 @@ protocol Endpoint {
 
 extension Endpoint {
 
-   private func urlComponents(
+   func urlComponents(
       queryItems: [URLQueryItem])
       -> URLComponents
    {
       var components = URLComponents(string: base)!
-      components.path = path
+      components.path = components.path.appending(path)
```

Comment:
This approach also works for base URLs that don't include a path in them (e.g. `https://api.anthropic.com`), so existing functionality of the lib is unchanged